docs/docs/1-Getting-Started/1-tictactoe.md

```diff
@@ -254,7 +254,7 @@ function onPlayerQuit(player, roomState) {
     // highlight-end
   }
   // highlight-next-line
-  return { joinable: false, finished: true };
+  return { state, joinable: false, finished: true };
```

Comment:
can you include this change in `examples/tictactoe` and possibly `examples/tictactoe-tutorial`

Reply:
I added it to examples/tictactoe but the tutorial does not have the solution programmed in. I left the tutorial as is.